spec/models/like/like_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe User, Like, type: :model do
+  subject { Like.new(author_id: 1, post_id: 1) }
+  before { subject.save }
+
+  it 'is valid with valid attributes' do
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a user_id' do
+    subject.author_id = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a post_id' do
+    subject.post_id = nil
+    expect(subject).to_not be_valid
+  end
+end
```

Comment:
- [ ] Kindly note that all custom methods in models should be tested also, Kindly make sure the `likes_counter` is tested here.